src/NzbDrone.Core/Download/Clients/Flood/Flood.cs

```diff
@@ -18,17 +18,20 @@ namespace NzbDrone.Core.Download.Clients.Flood
     public class Flood : TorrentClientBase<FloodSettings>
     {
         private readonly IFloodProxy _proxy;
+        private readonly IDownloadSeedConfigProvider _downloadSeedConfigProvider;
 
         public Flood(IFloodProxy proxy,
                         ITorrentFileInfoReader torrentFileInfoReader,
                         IHttpClient httpClient,
                         IConfigService configService,
                         IDiskProvider diskProvider,
                         IRemotePathMappingService remotePathMappingService,
+                        IDownloadSeedConfigProvider downloadSeedConfigProvider,
```

Comment:
Nitpick:  Move this to above ITorrentFileInfoReader, that way the last parameters all go to the base class.

Reply:
Done.